scrapy/commands/parse.py

```diff
@@ -191,6 +191,7 @@ def callback(response):
 
             # parse items and requests
             depth = response.meta['_depth']
+            response.request.callback = cb
```

Comment:
Nice, this looks like a good spot to reset the callback.

I am going to nit-pick once more here: I would rather reset the callback to its original value (the one stored in `response.meta`) and not to the function it resolved to (`cb`). This will only have an impact if the original value was `None`, but it ensures that `scrapy parse` will behave just like `scrapy crawl` should you check something like `if response.request.callback:` (for whatever reason you would do that :sweat_smile:).